concepts/05 Widgets/FileManager/00 Getting Started with File Manager/10 Create a File Manager.md

```diff
@@ -63,4 +63,224 @@ The following code creates the **FileManager** widget and adds it to your page.
         // ...
     ];      
 
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-file-manager id="fileManager"
+        [fileSystemProvider]="fileItems">
+        <dxo-permissions 
+            [create]="true"
+            [copy]="true"
+            [move]="true"
+            [delete]="true"
+            [rename]="true" >
+        </dxo-permissions>
+    </dx-file-manager>  
+
+    <!-- tab: app.component.ts -->
+    import { Component } from '@angular/core';    
+
+    @Component({
+        selector: 'app-root',
+        templateUrl: './app.component.html',
+        styleUrls: ['./app.component.css']
+    })
+
+    export class AppComponent {
+        fileItems = [{
+            name: "Documents",
+            isDirectory: true,
+            items: [{
+                    name: "Projects",
```

Comment:
Please fix indentation across all these code examples. General rule: each next nesting level should be indented by 4 spaces from the previous level.